backend/subroker/views.py

```diff
@@ -1,3 +1,392 @@
+        if request.user.is_authenticated:
+            try:
+                group = Group.objects.get(id=group_id)
+            except(Group.DoesNotExist) as e:
+                return HttpResponse(status=404)
+            group.delete()
+            return HttpResponse(status=200)
+        else:
+            return HttpResponse(status=401)
+        
+    else:
+        return HttpResponseNotAllowed(['GET', 'PUT', 'DELETE'])
+
+def group_add_user(request, group_id):
+    if request.method == 'PUT':
+        if request.user.is_authenticated:
+            new_member = request.user
+            try:
+                group = Group.objects.get(id=group_id)
+            except(Group.DoesNotExist) as e:
+                return HttpResponse(status=404)
+            if(group.current_people >= group.membership.max_people):
+                return HttpResponseBadRequest
+            group.members.add(new_member)
+            group.save()
+            member_arr = []
+            members = group.members.all()
+            for member in members:
+                member_arr.append(member.id)
+            response_dict = {
+                "id": group.id, 
+                "members": member_arr
+            }
+            return JsonResponse(response_dict, status=200)
+        else:
+            return HttpResponse(status=401)
+
+    elif request.method == 'DELETE':
+        if request.user.is_authenticated:
+            try:
+                group = Group.objects.get(id=group_id)
+            except(Group.DoesNotExist) as e:
+                return HttpResponse(status=404)
+            group.members.remove(request.user)
+            return HttpResponse(status=200)
+        else:
+            return HttpResponse(status=401)
+    else:
+        return HttpResponseNotAllowed(['PUT', 'DELETE'])
+#Content
+def content_list(request, content_id):
+    return HttpResponse("content search")
+
+def content_info(request, content_id):
+    if request.method == 'GET':
+        if request.user.is_authenticated:
+            try:
+                content = Content.objects.get(id=content_id)
+            except (Content.DoesNotExist) as e:
+                return HttpResponse(status=404)
+            user_arr = []
+            users = content.user_id.all()
+            for user in users:
+                user_arr.append(user.id)
+            return JsonResponse({
+                "id": content.id, 
+                "the_movie_id": content.the_movie_id,
+                "name": content.name,
+                "favorite_count": content.favorite_cnt,
+                "user_id": user_arr
+                }, status=200)
+        else:
+            return HttpResponse(status=401)
+
+def recommendation(request, user_id):
+    return HttpResponse("recommendation")
+
+def user_favorite_list(request):
+    if request.method == 'GET':
+        if request.user.is_authenticated:
+            fav_return = [content for content in Content.objects.filter(user_id__in=[request.user]).values()]
```

Comment:
@muhwagwa I think using User object is easier and more efficient(?) like below if related_name is "favorite_contents"
```suggestion
            fav_contents = [content for content in request.user.favorite_contents.all()]
```